webrtcclient.js

```diff
@@ -196,4 +196,33 @@ WebRTCClient.prototype.getStats = function() {
   });
 };
 
+WebRTCClient.prototype.waitForAudio(silence) {
```

Comment:
Not for this. The way all these steps are setup require communication between the nodejs controller and the browser via webdriver. Which would make looping very slow